apps/src/block_utils.js

```diff
@@ -1040,7 +1039,11 @@ exports.createJsWrapperBlockCreator = function(
               });
             }
           });
-          if (appOptions.level.miniToolbox && !appOptions.readonlyWorkspace) {
+          if (
+            !window.appOptions ||
```

Comment:
i don't understand the addition of `!window.appOptions` here -- why do we init mini-toolbox if we don't have appOptions?

(out of curiosity) are we moving away from using `/* global appOptions */` in favor of `window.appOptions`?

Reply:
This is for the levelbuilder block page which doesn't have `appOptions` but we do want to render the mini toolbox blocks.

Reply:
ah, right, that makes sense!

Reply:
(added a comment to the code for clarity)